enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -180,6 +180,20 @@ This controller shapes the static pod manifest itself.  It has some unique featu
     It waits until it is able to know that it is in the member list before trying to start.
     When it does this, it is able to determine the member list to use to launch.
 
+### Render Command
+Populates the static pod manifest used on the bootstrap node and other resource dependencies during the cluster bootstrap.
+
+#### File Reads
+1.) cluster-network-file - to determine the `ClusterCIDR` and `ServiceCIDR`. These values are used to conclude if the cluster is
+    single stack.
+2.) cluster-config-file - checks the declared `MachineCIDR` of the cluster from the install-config. Render uses the `MachineCIDR`
+    of the cluster to validate which IP address on the bootstrap interfaces is the BootstrapIP by making sure it is on the same
+    network as the `MachineCIDR`.
+
+#### File Writes
```

Comment:
add a debt section for pieces we need to move from their existing locations to here.  I think there are some MCO and installer rendered bits